Eatery/Controllers/BRB/BRBAccountManager.swift

```diff
@@ -0,0 +1,277 @@
+//
+//  AccountManager.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 10/24/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import WebKit
+
+struct HistoryEntry {
+
+    var description: String = ""
```

Comment:
I'm pretty sure you can just remove the equals empty string because it just be initialized to that by default.